### PR TITLE
Reserve an item ID range

### PR DESCRIPTION
**Pull request by Old-FW**

### CODE REVIEW

File: id-reservations/item-ids.json5 [hunk 1]
```diff
@@ -15,4 +15,5 @@
   [ -6000,  -6999, "AnnoyingPerson"],
   [ -7000,  -7999, "Cyph"],
   [-69000, -69999, "Penkly"],
+  [-8000, -8999, "Old-FW"],
```
Comment (raphendyr) [inline id-reservations/item-ids.json5]: Can you move this one line above, so the numbers are in order (-7000, -8000, -69000) and can you add a bit space so the commas allign and then it looks neat. Thank you!

**Review by raphendyr — CHANGES_REQUESTED** [review]
Thanks! This also allowed me to fix the automated verification. As it noticed too, the lines are not in descending order. I would appreciate, if you could fix that!

File: id-reservations/item-ids.json5 (hunk 1 above)
Comment (Old-FW) [reply id-reservations/item-ids.json5]: Think I've done it correctly now. Let me know if I need to change anything else. Many thanks.

File: id-reservations/item-ids.json5 [hunk 2]
```diff
@@ -14,5 +14,6 @@
   [ -5000,  -5999, "Captain_Wonders"],
   [ -6000,  -6999, "AnnoyingPerson"],
   [ -7000,  -7999, "Cyph"],
+  [ -8000,  -8999, "Old-FW"],
   [-69000, -69999, "Penkly"],
-]
+  ]
```
Comment (raphendyr) [inline id-reservations/item-ids.json5:19]: Thanks for the change. However, also this brace moved, which was not supposed to. The actual reservation line is perfect!

Comment (Old-FW) [reply id-reservations/item-ids.json5:19]: When I go to edit the file it looks good.

[
  // Legacy reservations, deprecated
  [8890100, 8890150, "Sinai"],
  [5850750, 5850750, "Sinai"],
  [2800900, 2800902, "Sinai"],
  [2501000, 2502999, "ehaugw"],

  // active reservations"
  [     0,   -999, "ehaugw"],
  [ -1000,  -1999, "IggyTheMad"],
  [ -2000,  -2999, "Saint"],
  [ -3000,  -3999, "raphendyr"],
  [ -4000,  -4999, "Kek"],
  [ -5000,  -5999, "Captain_Wonders"],
  [ -6000,  -6999, "AnnoyingPerson"],
  [ -7000,  -7999, "Cyph"],
  [ -8000,  -8999, "Old-FW"],
  [-69000, -69999, "Penkly"],
  ]

I don't know what to change here.

**Review by raphendyr — APPROVED** [review]

File: id-reservations/item-ids.json5 (hunk 2 above)
Comment (raphendyr) [reply id-reservations/item-ids.json5:19]: No problem. I merged this manually and fixed that :)

**Comment by raphendyr** [comment]
Thank you!

File: id-reservations/item-ids.json5 (hunk 2 above)
Comment (Old-FW) [reply id-reservations/item-ids.json5:19]: Thanks a lot.

**Pull request closed without merging**